src/js/views/patients/schedule/schedule_views.js

```diff
@@ -277,6 +284,9 @@ const ScheduleListView = CollectionView.extend({
   childViewEvents: {
     'render:children': 'onChildFilter',
   },
+  childViewTriggers: {
+    'click:patientSidebarButton': 'click:patientSidebarButton',
```

Comment:
Is there a better way to trigger `'click:patientSidebarButton'` than passing it up multiple times like I am here?

I'm handling it three times: `DayItemView` => `DayListView` => `ScheduleListView` (where the `schedule_app` is listening for it).

Reply:
there's a few ways we could do it..  this is one, but it is a lot of proxying.. could setup a radio channel that gets passed down as an intermediate.  However check out `onClickPatient` we're doing a radio request right there..  probably for this reason..  if there was more data things I'd probably have had that moved to the app, but you can add a `onClick...` here to match the others.  Probably worth it.

Reply:
Doing the `Radio.request()` in `DayItemView` works well. Updated in the latest commits I just pushed.